### ISSUE TITLE
Wrong version in 1.1.1 release

### ISSUE THREAD

**Issue by bastelfreak** [liufengyun/hashdiff#97]
Hi,
version 1.1.1 lists 1.1.0 as its version: https://github.com/liufengyun/hashdiff/blob/v1.1.1/lib/hashdiff/version.rb

Could you please do a 1.1.2 release with the correct version?

**Comment by liufengyun** [comment]
Thank you for the catch @bastelfreak . I'll propose a fix and a new release soon.

**Comment by liufengyun** [comment]
Fix here: https://github.com/liufengyun/hashdiff/blob/v1.1.2/lib/hashdiff/version.rb

Release: https://rubygems.org/gems/hashdiff/versions/1.1.2

**Comment by etiennebarrie** [comment]
It's just the Git tag that was incorrect, the released version of 1.1.1 did have the correct version:
```console
$ ruby -e 'gem "hashdiff", "1.1.1"; require "hashdiff"; puts Hashdiff::VERSION'                                                                                                                         
1.1.1
```
https://my.diffend.io/gems/hashdiff/1.1.0/1.1.1#d2h-807149

The tag should be on 5ad32a9c0a96d170e5f77c5114d9477d761307a6.

**Comment by liufengyun** [comment]
Thank you @etiennebarrie , I just force changed that tag (hopefully that wouldn't create troubles for people).
